neurogym/utils/data.py

```diff
@@ -55,12 +55,9 @@ def __init__(
             # TODO: infer sequence length from task
             seq_len = 1000
 
-        obs_shape = env.observation_space.shape
-        action_shape = env.action_space.shape
-        if len(action_shape) == 0:
-            self._expand_action = True
-        else:
-            self._expand_action = False
+        obs_shape = tuple(env.observation_space.shape)  # type: ignore[arg-type]
+        action_shape = tuple(env.action_space.shape)  # type: ignore[arg-type]
+        self._expand_action = len(action_shape) == 0
```

Comment:
this formulation is shorter than the previous one, but I find it harder to read. What was the reason for changing it?

an alternative, which I think is short and more readable (and allows for `action_shape` to be None) would be:

```suggestion
        self._expand_action = False if action_shape else True
```

or

```suggestion
        self._expand_action = not action_shape
```

Reply:
The reason for changing it is not for brevity but the fact that I don't like to rely on implicit conversion to `bool`. Is there a situation where `action_shape` would be `None`?